compiler/aarch64/runtime/ARM64arrayCopy.spp

```diff
@@ -159,20 +228,25 @@ fwByteCopy:
 finished:
 	ret
 
+adjustAddressForBackwardCopy:
+	add	x1, x1, x0
+	add	x2, x2, x0
+	b __backwardArrayCopy			// Branching to backward array copy after adjusting the address
```

Comment:
You can remove this `b` instruction.
Change the comment to `// Fall through to __backwardArrayCopy` or something similar in that case.